.env

```diff
@@ -39,3 +39,6 @@ ANALYTICS_URL=http://localhost:5000
 
 EXPERIMENTATION_KEY=6FA4C8D3DD2400B9D614BB7A229FD69B
 #GROWTHBOOK_CLIENT_KEY=override_me_in_env.development
+
+JWT_PUBLIC_KEY_PATH=../apps/.cert/public.pem
+JWT_PRIVATE_KEY_PATH=../apps/.cert/key.pem
```

Comment:
Feels a bit weird to link to other project like this 🤣

Reply:
Hehe yes quick and dirty, maybe we should think how to approach it down the line

Reply:
Yeah leave it for now.
We can actually simply move the certs to API I think.
Just means adjusting the scripts here  😅